mkdocs/docs/HPC/index.md

```diff
@@ -1,10 +1,32 @@
-# Welcome
+---
+hide: "toc"
+---
+# HPC Documentation
+
+---
+
+
+![image](img/ch0-hpc.png){: align=left style="height:383px;width:378px;border-radius:6px" }
+<!-- 1150 1134 -->
 
 Welcome to the {{ hpcinfra }} documentation!
 
 Use the menu on the left to navigate, or use the search box on the top right.
 
-The current view of the documentation is intended for people using **{{ OS }}** as operating system.
-To switch to different operating system (Windows, macOS, Linux), click [here](../#force_select_OS).
+You are viewing documentation intended for people using **{{ OS }}**. To switch to a different operating system (Windows, macOS, Linux), use the OS dropdown in the top bar.
+
+**Quick links**
+
+- [Getting Started](getting_started.md)  - [Getting Access](account.md)
+{%- if site == 'Gent' %}
+- [Recording of HPC-UGent intro](https://www.ugent.be/hpc/en/training/introhpcugent-recording) - [Linux Tutorial](./only/gent/linux-tutorial/index.md)
+- [Hardware overview](https://www.ugent.be/hpc/en/infrastructure)
+{%- endif %}
+- [FAQ](FAQ.md) - [Troubleshooting](troubleshooting.md) - [Best practices](./best_practices.md)
+- [Overview of available software]() <!-- TODO: add link -->
```

Comment:
Don't include this one yet, comment it out (or will be serving a landing page with a broken link, which will make people confused, or worse, make them open tickets)